accounting/transferwise/tasks.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+TransferWise tasks.
+"""
+
+from django.conf import settings
+from django.core.mail.message import EmailMessage
+from huey import crontab
+from huey.contrib.djhuey import db_periodic_task
+
+from accounting.transferwise import models
+
+
+@db_periodic_task(crontab(day=settings.TRANSFERWISE_BULK_PAYMENT_DAY, hour="0", minute="0"))
+def generate_bulk_payment_csv():
+    """
+    Send an email to the designated TransferWise Bulk Payment sender containing the generated bulk payment CSV.
+    """
+    # Make the new bulk payment for the sender.
+    bulk_payment = models.TransferWiseBulkPayment.objects.create()
+    bulk_payment.create_payments()
+    bulk_payment_csv = bulk_payment.to_bulk_payment_csv(bulk_payment.payments.all())
```

Comment:
PyCharm complains and says 'payments' isn't part of 'self' or something in the mixin class, which is annoying to see, but also technically true. But yeah, since we're still using the `TransferWisePayment` class' API in the mixin (by saying `payment.invoice`), it probably just makes sense to assume connection to something that has `payments` as a variable. So let's change it.